### ISSUE TITLE
Spellcheck popup suggestions are unreadable

### ISSUE THREAD

**Issue by MikhailKlemin** [texstudio-org/texstudio#137]
### Environment
- TeXstudio:   2.12.8 (hg n/a) 
- Qt: Using Qt Version 4.8.7, compiled with Qt 4.8.7 R
- OS: Ubuntu 16.04
- TeX distribution: TeX 3.14159265 (TeX Live 2015/Debian)

### Expected behavior
The Suggesitons should be redable

### Actual behavior
The spellchecker correcly marks misspelled words, but  sugesstions are messed up with encoding (see screen shot bellow)

### How to reproduce
Install TexStudio, install latest russian dictionary from LibreOffice [here](https://extensions.libreoffice.org/extensions/russian-spellcheck-dictionary.-based-on-works-of-aot-group )
Try to spell check "непровильное слово", you get that messed up popup. 

![image](https://user-images.githubusercontent.com/8426672/39702052-201e0844-520c-11e8-870b-ce56713c3874.png)

**Comment by sunderme** [comment]
the dict-file seems not to be encoed in utf-8. txs needs utf-8 encoded dictionaries.

**Comment by sunderme** [comment]
I have to correct myself. Works fine here. 
You have the .dic *and* the .aff file placed next to each other ?

**Comment by MikhailKlemin** [comment]
I have downloaded *.oxt file and fed it to the "Import Dictionary", then I selected `Default-Language: russian-aot`. Where to look for .dic and .aff ? In `/usr/share/texstudio`  nothing with `ru`

**Comment by thatlittleboy** [comment]
Try extracting the `.oxt` file using 7zip and then placing both `russian-aot.aff` and `russian-aot.dic` into the dictionaries folder instead.

**Comment by MikhailKlemin** [comment]
Same, but it seems like there is problem with dic file, I can't read it with text editor --  same messed up encoding.

**Comment by MikhailKlemin** [comment]
Funny thing is that I downloaded UTF-8 dictionaries from [here](https://code.google.com/archive/p/hunspell-ru/)  and it is still same unreadable popup! 
I dont know what is this, the spell checker picks misspelled words correcltly, but suggestions messed up.

**Comment by sunderme** [comment]
the point is the qt4 version of txs, use the qt5 version seems to work (you could install via the ppa)

**Comment by MikhailKlemin** [comment]
Yes, thank you. No need to PPA there is compileld deb, I do notknow why qt4 version was 1st and qt5 last, I just picked up what was 1st onlist for ubuntu 16.04. 

Spell check works as intended with qt5

**Comment by MikhailKlemin** [comment]
Just curious if the qt4 version no longer supported?
